components/ghost/ghost.app.js

```diff
@@ -0,0 +1,17 @@
+const axios = require("axios");
+
+module.exports = {
+  type: "app",
+  app: "ghost_org_content_api",
+  methods: {
+    async _makeRequest(endpoint, page) {
+      limit = 15;
```

Comment:
Declare `limit` as non-global, please (`const` or `let`)

Reply:
Updated to declare limit as const.